composer.json

```diff
@@ -51,6 +51,10 @@
             "vendor/bin/phpcs -p",
             "vendor/bin/psalm",
             "vendor/bin/phpunit"
+        ],
+        "post-install-cmd": [
```

Comment:
Instead of post-install, how about we utilize nuclei option: -template-url

Reply:
will do